ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/ExceptionsTest.kt

```diff
@@ -40,6 +41,88 @@ class ExceptionsTest : ClientLoader() {
         }
     }
 
+    @Test
+    fun testTextInResponseException() = testTextInException(
+        code = HttpStatusCode.BadRequest,
+        message = "Some request",
+        exceptionType = ResponseException::class
+    )
+
+    @Test
+    fun testTextInRedirectResponseException() =
+        testTextInException(
+            code = HttpStatusCode.PermanentRedirect,
+            message = "Some redirect",
+            exceptionType = RedirectResponseException::class
+        )
+
+    @Test
+    fun testTextInClientRequestException() =
+        testTextInException(
+            code = HttpStatusCode.Conflict,
+            message = "Some conflict",
+            exceptionType = ClientRequestException::class
+        )
+
+    @Test
+    fun testTextInServerRequestException() =
+        testTextInException(
+            code = HttpStatusCode.VariantAlsoNegotiates,
+            message = "Some variant",
+            exceptionType = ServerResponseException::class
+        )
+
+    @Test
+    fun testBinaryGarbageInExceptionMessage() = testTextInException(
+        code = HttpStatusCode.BadRequest,
+        message = """
+            .Q
+            build.gradle�Q�J�0��+B��NwŃ<�A����C7m��${'$'}]���n�v��������7o&��+��B�4�]���^�,����
```

Comment:
Shouldn't we escape this garbage?

Reply:
Why should we?
It's a test for a garbage string.